trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/plugins/generator/cacsl2boogietranslator/preferences/CACSLPreferenceInitializer.java

```diff
@@ -61,11 +62,12 @@ public class CACSLPreferenceInitializer extends UltimatePreferenceInitializer {
 	public static final String LABEL_CHECK_MEMORY_LEAK_IN_MAIN =
 			"Check for the main procedure if all allocated memory was freed";
 	public static final String LABEL_SVCOMP_MEMTRACK_COMPATIBILITY_MODE = "SV-COMP memtrack compatibility mode";
-	public static final String DESC_SVCOMP_MEMTRACK_COMPATIBILITY_MODE = "Report UNKNOWN instead of UNSAFE if not all "
-			+ "allocated memory was freed at the end of the main procedure. Rationale: at the SV-COMP we have to check "
-			+ "if the program lost track of allocated memory. If this is set to false we are unsound (at SV-COMP) in "
-			+ "cases where not all memory is freed but pointers to that memory are live at the end of the "
-			+ "main procedure.";
+	public static final String DESC_SVCOMP_MEMTRACK_COMPATIBILITY_MODE = """

```

Comment:
While I think these multiline Strings are generally nicer, I don't think we should change existing code, as also the line breaks are somehow strange now.